test-suite/output/notation_prefix_incompatible_level.out

```diff
@@ -0,0 +1,9 @@
+File "./output/notation_prefix_incompatible_level.v", line 2, characters 0-43:
+Warning: Notations "#0 #1" defined at level 30 and "#0 #1 #2"
+defined at level 40 have incompatible prefixes.
```

Comment:
```suggestion
File "./output/notation_prefix_incompatible_level.v", line 2, characters 0-43:
Warning: Notations "#0 #1" defined at level 30 (in File "foobar", line 1, 0-40) and "#0 #1 #2"
defined at level 40 have incompatible prefixes.
```

This message would be better if it gave the location of the first notation.  Otherwise the user has to search/scan for the definition, which could in be many places.

Reply:
Yes, in fact we could need the locations of the two notations because the warning can also be raised when importing two incompatible modules.
However, this is clearly outside the scope of the current PR. IIRC there were already attempts by @ejgallego to locate things but this raised interesting questions and did not converge.

Reply:
Surely it can't be that hard to add a Loc.t to the notation data structure, but it's your call.

Reply:
Well, prior experiences show it is definitely non trivial.